src/csharp/Grpc.IntegrationTesting/ClientRunners.cs

```diff
@@ -55,6 +58,15 @@ public class ClientRunners
     {
         static readonly ILogger Logger = GrpcEnvironment.Logger.ForType<ClientRunners>();
 
+        // Profilers to use for clients.
+        static readonly BlockingCollection<BasicProfiler> profilers = new BlockingCollection<BasicProfiler>();
+
+        internal static void AddProfiler(BasicProfiler profiler)
```

Comment:
High level comment:
Perhaps I don't fully understand the use case of profiling, but could we simplify this by just instantiating a profiler in ClientRunnerImpl if some boolean field is set?  Or do we actually need to be able to set a specific profiler per client runner?